CMakeLists.txt

```diff
@@ -872,9 +873,9 @@ target_link_libraries(nheko PRIVATE
 	MatrixClient::MatrixClient
 	cmark::cmark
 	spdlog::spdlog
-	Qt::Widgets
+        Qt::Widgets
 	Qt::Svg
-	Qt::Gui
+        Qt::Gui
```

Comment:
You can probably leave out those changes :p

Reply:
Oh yes. At Creator loves converting tabs to spaces.